rho/scancommand.py

```diff
@@ -324,9 +323,7 @@ def run_ansible_with_vault(cmd_string, vault_pass, ssh_key_passphrase=None,
             if child.isalive():
                 child.wait()
             if log_to_stdout:
-                # tail will kill itself once it is done copying data
-                # to stdout, thanks to the --pid option.
-                tail.wait()
+                tail.terminate()
```

Comment:
I'm mostly worried about the output of `redhat-packages`. I know that solution is ugly, but I don't know a better solution that isn't a whole lot more complex.